llvm/lib/Transforms/Vectorize/VPRecipeBuilder.h

```diff
@@ -102,6 +102,12 @@ class VPRecipeBuilder {
   VPWidenRecipe *tryToWiden(Instruction *I, ArrayRef<VPValue *> Operands,
                             VPBasicBlock *VPBB);
 
+  /// Makes Histogram count operations safe for vectorization, by emitting a
```

Comment:
Maybe this can be generalized a bit more, so it doesn't refer to how AArch64 will be doing this?

Maybe something like:

```
/// Makes Histogram operations safe for vectorization, by emitting a
/// Histogram LLVM Intrinsic in place of the Load / BinOp (Add/Sub) / Store
/// operations that do the actual histogram computation.
```